docs/general/license/activation.md

```diff
@@ -5,9 +5,99 @@ sidebar_position: 3
+
+**Below you can check step-by-step activation instructions for each platform.**
+
+### Activating AdGuard for Windows
+
+1. Open the AdGuard app
+   
+To do it, doubleclick on the AdGuard icon on the desktop:   
+
+![AdGuard icon on the desktop](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowsEn.png)
+
+or double click on the AdGuard icon in the system tray:   
+
+![AdGuard icon in the system tray](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windows2En.png)
+
+2. Go to **License** in the main menu, then click **Activate license** 
+
+![Click Activate license](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense1en.png)
+
+3. You will be presented with two options for activation:
+
+- Via entering **Email** and **Password** from your **AdGuard  account** (or through a FB/Google/Apple account)
+
+![Log in to your AdGuard account](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense2en.png)
+
+- Via entering the **license key**
+
+![Enter license key](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense3en.png)
+
+### Activating AdGuard for Mac
+
+1. Open AdGuard app (lick on the AdGuard icon in the menu bar) 
+
+![AdGuard icon in the menu bar](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Ja/General/mac1.png)
+
+2. Click on **gear icon** in the top right corner
+3. Click **License...** 
+
+![Go to Settings and License](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/macEn.png)
+
+4. In the opened window press **Activate** button   
+
+![Press Activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen1.png)
+
+5. You will be presented with two options for activation:
+- Activate by entering **Email** and **Password** from your **AdGuard  account** (or through a FB/Google/Apple account)
+
+![Log in to your AdGuard account to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen2.png)
+
+- Activate with a **license key**
+
+![Enter your license key to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen3.png)
+
+### Activating AdGuard for Android
+
+1. Open the AdGuard app
+2. Tap **menu icon (☰)** in the top left corner of the screen
+3. Tap **Get Premium** bar (or tap **License Status**)
+
+![Tap Get Premium or License Status](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/androidlicense1en.png)
+
+4. Tap **three dots menu (⋮)** in the top right corner of the screen
+
+![Tap three dot menu](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/android2En.png)
+
+5. Press **Activate Premium**, and you will be presented with two activation options:
+
+- Enter **Email** and **Password** from your **AdGuard account** (or sign in with Facebook/Google account)
```

Comment:
Via entering the email and password from your AdGuard account (or from your Facebook, Google, or Apple account)